ansible/library/exabgp.py

```diff
@@ -203,14 +206,14 @@ def remove_exabgp_supervisord_conf(name):
     except Exception:
         pass
 
-def setup_exabgp_processor():
+def setup_exabgp_processor(default_dump_script):
     try:
         os.mkdir("/usr/share/exabgp", 0755)
     except OSError:
         pass
-
-    with open("/usr/share/exabgp/dump.py", 'w') as out_file:
-        out_file.write(dump_py)
+    if default_dump_script:
+        with open(DEFAULT_DUMP_SCRIPT, 'w') as out_file:
+            out_file.write(dump_py)
```

Comment:
I don't understand the logic. Where we get dump.py here?

Reply:
I added a new dump script ```tests/bgp/bgp_monitor_dump.py``` which will parse the json output of exabgp and write into a dump file. The script is copied to PTF in test case logic when we pass ```dump_script``` argument to exabgp.
```
 ptfhost.copy(src=CUSTOM_DUMP_SCRIPT, dest=CUSTOM_DUMP_SCRIPT_DEST)
```